lib/index.js

```diff
@@ -75,6 +76,7 @@ function getFooterNode() {
  * @param {Boolean} [options.routing=false] Specify whether to request additional metadata about the recommended navigation destination corresponding to the feature or not. Only applicable for address features.
```

Comment:
@mpothier set of parameters could be different. For example we have `routing` and `fuzzyMatch` in v5, and have `permanent` parameter in v6